doc/source/whatsnew/v0.17.0.rst

```diff
@@ -354,15 +362,15 @@ Some East Asian countries use Unicode characters its width is corresponding to 2
 
 .. ipython:: python
 
-   df = pd.DataFrame({u'国籍': ['UK', u'日本'], u'名前': ['Alice', u'しのぶ']})
+   df = pd.DataFrame({u"国籍": ["UK", u"日本"], u"名前": ["Alice", u"しのぶ"]})
    df;
 
 .. image:: ../_static/option_unicode01.png
 
 .. ipython:: python
 
-   pd.set_option('display.unicode.east_asian_width', True)
-   df;
+   pd.set_option("display.unicode.east_asian_width", True)
+   df
```

Comment:
Is semi-colon need to be added here? @dsaxton @MarcoGorelli

Reply:
Yeah I think we want one here too